tests/CMakeLists.txt

```diff
@@ -37,4 +37,5 @@ endif()
 include_directories(${GTEST_INCLUDE_DIRS})
 target_link_libraries(dummy_test ${GTEST_BOTH_LIBRARIES})
 
-add_subdirectory(carddatabase)
\ No newline at end of file
+add_subdirectory(carddatabase)
+add_subdirectory(loading_from_clipboard)
```

Comment:
This is fine for now but I wonder how we can collect all these to one binary later - I'm looking to set up code coverage and the stuff I've found works better with a single target.